cli/run/index.ts

```diff
@@ -67,6 +67,12 @@ export default async function runRollup(command: any) {
 				for (const inputOptions of options) {
 					await build(inputOptions, warnings, command.silent);
 				}
+				if (command.failAfterWarnings && warnings.total) {
+					throw {
+						code: 'FAIL_AFTER_WARNINGS',
+						message: 'Warnings occurred and --failAfterWarnings flag present'
+					};
```

Comment:
throwing the object felt a bit strange. Not sure if having a

```ts
warnings.flush();
handleError({
  code: 'FAIL_AFTER_WARNINGS',
  message: 'Warnings occurred and --failAfterWarnings flag present'
});
```
would be better?

Reply:
Throwing something that is not an `Error` instance is always a bad idea IMO as it violates expectations. `handleError` is the correct way to do it.